data/operators/leisure/park.json

```diff
@@ -1395,6 +1395,25 @@
         "operator:wikidata": "Q81824"
       }
     },
+    {
+      "displayName": "City of Houston",
+      "id": "houstonparksandrecreationdepartment-8bfd34",
+      "locationSet": {
+        "include": ["us-tx.geojson"]
+      },
+      "tags": {
+        "access": "public",
+        "addr:state": "TX",
+        "leisure": "park",
+        "operator": "Houston Parks and Recreation Department",
+        "operator:phone": "+1-832-395-7000",
```

Comment:
```suggestion
```
Please note that we don't include `:phone` or other contact tags in NSI. Instead, we encourage people to add this information to their respective Wikidata item. (see https://github.com/osmlab/name-suggestion-index/wiki/Judge-Case#can-we-add-websitephonewikipedia)

Reply:
That's an interesting design decision. So we have to add the data to Wikidata, which the search/index/retrieval functionality does not support and that effectively locks the data in Wikidata in perpetuity because of licensing mismatch? I mean it would make sense to have something like Wikidata which OSM could store this data about discrete ways/pois in a centralized location. But it doesn't make much sense when the suggestion is to put the data somewhere none of the tooling currently looks (server or client) and where it can't be imported in the future.

For example, is there a plan to support search by phone-by-number in OSM? That's incredibly useful for linking a phone book to GIS.

Reply:
> that effectively locks the data in Wikidata in perpetuity because of licensing mismatch?

[Wikidata is licensed under CC0 (public domain](https://www.wikidata.org/wiki/Wikidata:Licensing)), so I don't think licensing will be an issue.

> But it doesn't make much sense when the suggestion is to put the data somewhere none of the tooling currently looks (server or client)

That's a good point, that was part of the reason why I opened #5275. This decision was mostly based on https://github.com/osmlab/name-suggestion-index/issues/5275#issuecomment-905798587

Reply:
NSI is designed to help make OSM tagging consistent for the things that mappers would be tagging anyways. It isn’t designed to be a general-purpose vehicle for information tangential to a feature on the ground.

> But it doesn't make much sense when the suggestion is to put the data somewhere none of the tooling currently looks (server or client) and where it can't be imported in the future.

Many OSM data consumers, [including Nominatim](https://nominatim.org/release-docs/3.5.1/data-sources/Wikipedia-Wikidata/), already cross-reference Wikidata for various things, just not phone numbers specifically.

> For example, is there a plan to support search by phone-by-number in OSM? That's incredibly useful for linking a phone book to GIS.

Nominatim has declined to add support for searching by phone number, let alone searching by the operator’s phone number: osm-search/Nominatim#861. I don’t think it has been proposed for Pelias so far. In any event, a query tool would be a more appropriate tool for any use case requiring an exact string match to a unique identifier. Indeed, the Overpass API already supports querying for features by `phone` or any other key.

Reply:
@arch0345 Just to follow up on the licensing question,

> Wikidata is using the CC0 license, which on first sight allows imports into OSM. However, OSM does not, generally, accept imports of data from Wikidata due to the sources often used to contribute to Wikidata. [...] OpenStreetMap tries to be whiter than white, and data should be legally used in any country[[4]](https://wiki.openstreetmap.org/wiki/Wikidata#cite_note-fairhurst-4). Therefore OpenStreetMap acknowledges [sui generis database right](https://en.wikipedia.org/wiki/en:Sui_generis_database_right). As a conclusion, an import of information from Wikidata into OSM **is generally not permitted because Wikidata has lower standards in terms of copyright than OSM**. In addition, Wikidata contains some data from sources we are not allowed to use[[5]](https://wiki.openstreetmap.org/wiki/Wikidata#cite_note-koppenhoefer-5).